src/components/ObitPane/ObitPane.js

```diff
@@ -27,43 +27,46 @@ const ObitPane = ({obit}) => {
     }
     
     return (
-        <section className='obit'>
-            <div className='small-square corner1'></div>
-            <div className='rectangle'></div>
-            <div className='small-square middle1'></div>
-            <div className='rectangle'></div>
-            <div className='small-square corner2'></div>
-            <div className='rectangle'></div>
-            <div className='rectangle'></div>
-            <div className='small-square middle2'></div>
-            <div className='small-square middle3'></div>
-            <div className='rectangle'></div>
-            <div className='rectangle'></div>
-            <div className='small-square corner3'></div>
-            {editMode 
-                ? <button className='edit-button rectangle' onClick={() => handleSubmit()}>Submit</button>
-                : <button className='edit-button rectangle' onClick={() => handleClickEdit()}>Edit</button>
-            }
-            <div className='small-square middle4'></div>
-            {editMode
-                ? <div className='rectangle'><p className='remaining'>Limit: {usedChars}/500</p></div> 
-                : <div className='rectangle'></div>
-            }
-            <div className='small-square corner4'></div>
-            {error && <div className='obit-error'>{error}!</div>}
-            {editMode 
-                ?   <textarea 
-                        className='obit-text'
-                        placeholder='How do you want to be remembered?'
-                        value={obituary}
-                        onChange={(e) => {handleChange(e)}}
-                        maxLength='500'
-                    >
-                    </textarea>   
-                    
-                : <article className='obit-text'>{obituary}</article>
-            }
-        </section>
+        <>
```

Comment:
What do these empty carrots do?

Reply:
This is called a fragment - basically this is a JSX element that doesn't appear on the page, but it still allows the elements to be grouped together and returned as a unit to the virtual DOM.